packages/design-system/src/components/section-title.tsx

```diff
@@ -0,0 +1,174 @@
+/**
+ * Implementation of the "Title (title of: section)" component from:
+ * https://www.figma.com/file/sfCE7iLS0k25qCxiifQNLE/%F0%9F%93%9A-Webstudio-Library?node-id=2%3A12361
+ */
+
+import { PlusIcon } from "@webstudio-is/icons";
+import {
+  forwardRef,
+  type ReactNode,
+  type Ref,
+  type ComponentProps,
+} from "react";
+import { theme, css, type CSS } from "../stitches.config";
+import { Button } from "./button";
+import { textVariants } from "./text";
+import { cssVars } from "@webstudio-is/css-vars";
+import { handleArrowFocus } from "./primitives/arrow-focus";
+
+const addIconColor = cssVars.define("add-icon-color");
+
+const containerStyle = css({
+  position: "relative",
+  height: theme.spacing[15],
+  [addIconColor]: theme.colors.foregroundSubtle,
+  "&:hover, &:has(:focus-visible), &[data-state=open]": {
+    [addIconColor]: theme.colors.foregroundMain,
+  },
+});
+
+const buttonStyle = css({
```

Comment:
hmm? isn't this supposed to be our button component?

Reply:
This is not our button. This is just styles for `<button>` to make it take entire space of the title. This is what you click when you click on the title.

Reply:
damn, ok lets call it titleButtonStyle, because I totally thought its the + button there